src/backend/database_migrations/versions/20220114_192608_set_autoubuild_tree_user.py

```diff
@@ -0,0 +1,45 @@
+"""set autoubuild tree user
+
+Create Date: 2022-01-14 19:26:10.048670
+
+"""
+import enumtables  # noqa: F401
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "20220114_192608"
+down_revision = "20220103_132500"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+
+    # Create a new bot group and add our bot user to it.
+    create_group_sql = sa.sql.text(
+        "INSERT INTO aspen.groups (name, address, prefix, default_tree_location_id) SELECT 'Bot Users', 'Bot Users', 'bot', id FROM aspen.locations WHERE division = 'California' AND location = 'San Mateo County'"
```

Comment:
id doesn't need to be in quotes?

Reply:
the fields that we've quoted in this query are hard-coded strings - we want to use `Bot Users` as the group name, but we want to use the dynamic `id` value we select from the locations table to populate the group's location.